ord_schema/dataset_templating.py

```diff
@@ -52,12 +52,13 @@
 flags.DEFINE_boolean('validate', True, 'If True, validate Reaction protos.')
 
 
-def read_spreadsheet(file_name):
+def read_spreadsheet(file_name_or_buffer, suffix=None):
```

Comment:
Have you tested this with a buffer? I don't know how the calls to `os.path.splitext` work in that case.

Reply:
Yes; see the tests in #327. If you try to call this with a buffer and don't provide `suffix` you should get an exception.